crates/cheatnet/tests/common/assert.rs

```diff
@@ -0,0 +1,21 @@
+#[allow(clippy::module_name_repetitions)]
+#[macro_export]
+macro_rules! assert_success {
+    ($call_contract_output:expr,$expected_data:expr) => {
+        assert!(match $call_contract_output {
```

Comment:
this, and one below should be `assert_eq!` imho

Reply:
Not sure how to go about this if we also want to assert it's variant